plugin/metrics/prometheus/factory_test.go

```diff
@@ -60,12 +60,16 @@ func TestWithConfiguration(t *testing.T) {
 	err := command.ParseFlags([]string{
 		"--prometheus.server-url=http://localhost:1234",
 		"--prometheus.connect-timeout=5s",
+		"--prometheus.allow-token-from-context=true",
+		"--prometheus.token-file-path=testdir/test_file.txt",
```

Comment:
Please add a test case where the file path contains a space (perfectly valid, but unconventional).